clients/web/src/lib/components/atoms/ProgressBar/ProgressBar.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    // Obviously, this needs to be its own parameter for the 'filled-ness' of
+    // the bar
+    export let progress = 45;
+
+    // Props to customize which bar you want
+    export let size = "default";
+    export let label = false;
+    export let labelOutside = true;
+    export let progressLabel = "Default Progress Bar Text";
+
+    // Styling choices
+    export let primaryBarColor = "bg-primary-900";
+    export let primaryTextClass = "text-primary-200";
+</script>
+
+{#if label}
+    {#if labelOutside}
+        <div class="flex justify-between mb-1">
```

Comment:
In this case we should also be able to move `{primaryTextClass}` (or the variant class) to the root elements; should prevent repetition

Reply:
Resolved